frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -1135,6 +1144,24 @@ function ReaderDictionary:downloadDictionary(dict, download_location, continue)
     end
 end
 
+function ReaderDictionary:extendIfoWithLanguage(dict, download_location)
+    local function cb(path, filename)
+        if util.getFileNameSuffix(filename) == "ifo" and dict.ifo_lang then
```

Comment:
(Dunno if in the future we will want to add other things than ifo_lang - but you check dict_ifo_lang last - if you checked it first on line 1132, you could avoid calling extendIfoWithLanguage() when it is not set. But ok, it's now always set :)

Reply:
good point, I did call it extendIfo**WithLanguage** specifically so I might as well check it up front. :smile: